### DISCUSSION TITLE
There is NO Data shown in the Alerts, Hunts, and Zeek Restarts via Healthcheck!

### DISCUSSION THREAD

**Discussion by M3reth** [Security-Onion-Solutions/securityonion#7575] [Unsupported Versions] [answered]
Hi guys. I have installed Security Onion and it has generated some problems. Currently using version "2.3.61". I have already used the "soup" command to update the version, but it does not update to the latest version, but this is not the problem that is troubling me. 

Since its standalone installation, the SOC page does not show data in the Alerts, Hunt, and Zeek Restarts via Healthcheck section, but the signatures are recorded in /nsm/suricata/.  

When generating the So-status command it shows all running services "OK" also in the grid section it is OK.

Attached is the output of the command "salt \* so.status". 

Thanks for your support!
[salt.txt](https://github.com/Security-Onion-Solutions/securityonion/files/8256902/salt.txt)

**Comment by dougburks** [comment, marked as answer]
You can check `/opt/so/log/` for additional clues.

However, [version 2.3.61 is from July of last year](https://blog.securityonion.net/2021/07/security-onion-2361-now-available.html) and that version has several security vulnerabilities. You'll want to update as soon as possible and it may be quickest and easiest to simply wipe and install the latest version.

**Reply by M3reth** [reply]
Hello Mr. dougburks. 
First of all, thanks for your help. i have already updated to version 2.3.110.

now when I run the command " salt \* state.highstate" it generates me the following output:

#Failed to compile data:
----------
    The function "state.highstate" is running as PID 9498 and was started on 2022, Mar 16 19:30:36.128383 with jid 20220316193036128383

ERROR: Minions returned with non-zero exit code

**Reply by dougburks** [reply]
> The function "state.highstate" is running as PID 9498 and was started on 2022, Mar 16 19:30:36.128383 with jid 20220316193036128383

This is most likely nothing to worry about. It just means that the scheduled highstate kicked off right before you started your highstate.

> ERROR: Minions returned with non-zero exit code

This happens when you run `salt *` on a standalone installation. So instead of `salt * state.highstate`, try running `salt-call state.highstate`.

**Reply by M3reth** [reply]
Hello Mr. dougburks. 

Thanks for the support, with the command "salt-call state.highstate" I managed to generate the attached summary. 

In this summary it generated two bugs which I mention below (The complete summary in the attached file):

1:
"ID: wait_for_elasticsearch
    Function: cmd.run
        Name: so-elasticsearch-wait
      Result: False
     Comment: Command "so-elasticsearch-wait" run
     Started: 21:19:07.428165
    Duration: 342215.186 ms
     Changes:   "

2: ID: so-elastalert
    Function: docker_container.running
      Result: False
     Comment: One or more requisite failed: elastalert.wait_for_elasticsearch
     Started: 21:24:49.663782
    Duration: 0.006 ms
     Changes:   

Thank you. 
[salt.txt](https://github.com/Security-Onion-Solutions/securityonion/files/8307572/salt.txt)

**Reply by dougburks** [reply]
Was this a fresh installation of 2.3.110?

What options did you choose during Setup?

What are the hardware specs of the box?

**Comment by M3reth** [comment]
Hello Mr. dougburks.
This box was installed from the beginning of the new version of security onion 2 and since then, it did not show signatures in the alerts section. Just a week ago it was updated to version 2.3.110.

For the installation, the options were chosen:
- For the architecture we chose: Standalone 
- For internet connection: Direct
-Update mode - automatic
- Data capture - Zeek and Suricata.

The hardware specifications of the box:
- Intel(R) Xeon(R) CPU E3-1220 v3 @ 3.10GHz
- Thread(s) per core: 1
- Core(s) per socket: 4
- CPU family: 6
- RAM 16 G
- hard disk 1t

Thank you!

**Comment by dougburks** [comment]
If you want to troubleshoot the problem, you can check `/opt/so/log/` for additional clues and maybe perform a fresh installation (perhaps in a VM) to compare the systems side-by-side.

It may be quickest and easiest just to perform a fresh installation on the hardware itself.
